src/Handlers/CrawlRequestFulfilled.php

```diff
@@ -2,10 +2,12 @@
 
 namespace Spatie\Crawler\Handlers;
 
+use GuzzleHttp\Psr7\Uri;
 use Spatie\Crawler\Crawler;
 use Spatie\Crawler\CrawlUrl;
 use Spatie\Crawler\LinkAdder;
```

Comment:
I don't like this import ordering method. :disappointed:

Reply:
Alphabetical is the correct order for PSR-2 compliant code, so that's what gets enforced.

Reply:
I thought order by length was being used because no styleci errors adns watching the original file:
https://github.com/spatie/crawler/blob/e991498972a2e41daac0f12ba47346c2b9922ac2/src/Handlers/CrawlRequestFulfilled.php#L5-L13

I will fix it tomorrow ASAP.